device-types/Cisco/ASR1004.yaml

```diff
@@ -0,0 +1,29 @@
+---
+manufacturer: Cisco
+model: ASR1004
+slug: cisco-asr1004
+part_number: ASR1004
+comments: '[Cisco ASR 1004 Router](https://www.cisco.com/c/en/us/products/routers/asr-1004-router/index.html)'
+u_height: 4
+weight: 31.16
+weight_unit: kg
+front_image: true
+rear_image: true
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: Con 0
+    type: rj-45
+  - name: Aux 0
+    type: rj-45
+power-ports:
+  - name: ASR1004-PWR-AC-1
```

Comment:
Looks like these PSUs are modules, therefore they need to be modeled as `module-bays`